src/panels/energy/energyCalculator.ts

```diff
@@ -9,7 +9,7 @@ export const calculateEnergyFigures = (values, floorArea: number) => {
   const alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("");
 
   const ob = values.reduce((acc, curr, i) => {
-    curr.map((col, j) => {
+    curr.forEach((col, j) => {
```

Comment:
This was for the linter in case you're wondering

Reply:
Haha the linter probably vomited when it saw the code I've been throwing in for the past 24 hours